libs/staging-provider/src/Provider/StorageApiWorkspace.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StagingProvider\Provider;
+
+use Keboola\StagingProvider\Exception\StagingProviderException;
+use Throwable;
+
+readonly class StorageApiWorkspace
```

Comment:
Proc ne jen `Workspace`?

Reply:
Protože to bylo takový nejednoznačný, tohle je reprezentace odpovědi ze storage API, tak mě to takhle přišlo lepší